modules/md5sum/main.nf

```diff
@@ -0,0 +1,35 @@
+process MD5SUM {
+    tag "$meta.id"
+    label 'process_low'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bcl-convert. Please use docker or singularity containers."
```

Comment:
Wouldn't this message be a little bit misleading? I mean these modules could be used to check other files apart from the ones converted by bcl-convert

Reply:
This was a copy paste error, should already been fixed!